pki/generate_cert_and_key.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
This file with very opinionated and even contains personalized details does not belong to the repo.

A link to external documentation on how to generate the certificates in the example would be more suitable and sufficient. This is not specific for Modbus.